<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_table</class>
 <widget class="QWidget" name="Change_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор матрицы исходных данных</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_save">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить изменения</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
